package.json

```diff
@@ -65,36 +65,35 @@
     "cypress": "^9.4.1",
     "cypress-file-upload": "^5.0.8",
     "dayjs": "^1.10.7",
-    "ember-ajax": "^4.0.1",
     "ember-attacher": "git+https://github.com/erikap/ember-attacher.git#7697e098c911a5224557ec464d176f600dfe1d2f",
     "ember-auto-import": "^2.0.0",
-    "ember-cli": "~3.25.3",
-    "ember-cli-app-version": "^4.0.0",
+    "ember-cli": "~3.28.5",
+    "ember-cli-app-version": "^5.0.0",
     "ember-cli-autoprefixer": "^1.0.0",
     "ember-cli-babel": "^7.26.11",
     "ember-cli-cjs-transform": "^2.0.0",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-dependency-lint": "^1.1.3",
     "ember-cli-head": "^2.0.0",
-    "ember-cli-htmlbars": "^5.3.2",
-    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-htmlbars": "^6.0.1",
+    "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-moment-shim": "^3.7.1",
     "ember-cli-sanitize-html": "^2.0.3",
     "ember-cli-sass": "^8.0.1",
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.1",
+    "ember-cli-terser": "^4.0.2",
     "ember-cli-update": "^1.0.0",
     "ember-click-outside": "^2.0.0",
     "ember-code-snippet": "^3.0.0",
     "ember-composable-helpers": "^4.4.1",
     "ember-concurrency": "^2.0.2",
     "ember-concurrency-decorators": "^2.0.3",
-    "ember-data": "~3.25.0",
+    "ember-data": "~3.28.6",
```

Comment:
I think we really want to make sure we have 3.28.8 or up because of this https://github.com/emberjs/data/pull/7805#issuecomment-1028568379